tool/firebase_util.py

```diff
@@ -0,0 +1,13 @@
+import enum
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+def init(CRYPTO_TOKEN_PATH, log):
```

Comment:
nit: rename this to `init_firebase_client` for clarity on what's being initialised, and we may also also want to add other functions here in the future